test/unit/featureUtils.js

```diff
@@ -12,13 +12,15 @@ describe('FeaturesUtils', function () {
     it('should correctly parse geojson', () =>
         promise.then((collection) => {
             assert.equal(collection.features.length, 3);
+            collection.updateMatrix();
```

Comment:
Is it mandatory to test the `updateMatrix` method ? If the object returned by the `GeoJsonParser.parse` method has the property tested above (`collection.features.lenght`), that means it must be an instance of `FeatureCollection`. Therefore, it also is an instance of `Three.Object3d`. Testing the `updateMatrix` method appears to be testing Threejs.

Moreover, this `collection` object is not used later in the scope of the `promise.then`, so I don't get the point of this line.

Am I missing something ?